packages/components/dev/App.tsx

```diff
@@ -109,148 +28,88 @@ const App = () => {
-                  )}
-                </For>
-              </ComboBox.Options>
-            </ComboBox>
-          </div>
-        </div>
-
-        <div class="mb-10">
-          <div>
-            <h2>Pharmacy Search</h2>
-            <PharmacySearch setPharmacy={setPharmacy} />
-            <h2>Pharmacy Search Initialized with Address</h2>
-            <PharmacySearch setPharmacy={setPharmacy} address="11221" />
-            <h2>Pharmacy Search set with Address after 2 seconds</h2>
-            <PharmacySearch setPharmacy={setPharmacy} address={timedAddress()} />
-          </div>
-        </div>
-
-        <div class="mb-10">
-          <div>
-            <h2>Dose Calculator</h2>
-            <Button onClick={() => setDoseOpen(true)}>
-              <Icon name="calculator" size="sm" />
-            </Button>
-
-            <DoseCalculator
-              open={doseOpen()}
-              onClose={() => setDoseOpen(false)}
-              setAutocompleteValues={({ liquidDose, totalLiquid, unit }) => {
-                console.log(liquidDose, totalLiquid, unit);
-              }}
-            />
-          </div>
-        </div>
-
-        <h2>Pharmacy Select</h2>
-        <Card>
-          <PharmacySelect
-            patientIds={patientIds()}
-            enableLocalPickup
-            enableSendToPatient
-            setFufillmentType={(t) => console.log('fulfillmentType: ', t)}
-            setPharmacyId={(p) => console.log('pharmacyId: ', p)}
-          />
-        </Card>
-        <h4 class="mt-8">With Mail Order</h4>
-        <Card>
-          <PharmacySelect
-            patientIds={['pat_01JEVF5DWQAQFHTVYK9ABG65JZ']}
-            enableLocalPickup
-            enableSendToPatient
-            mailOrderPharmacyIds={[
-              'phr_01GA9HPVBVJ0E65P819FD881N0',
-              'phr_01GCA54GVKA06C905DETQ9SY98'
-            ]}
-            address="11221"
-            setPharmacyId={(p) => console.log('pharmacyId! ', p)}
-            setFufillmentType={(t) => console.log('fulfillmentType! ', t)}
-          />
-        </Card>
-
-        <h2>Toast</h2>
-        <Button
-          onClick={() =>
-            triggerToast({
-              status: 'success',
-              header: 'What a great job',
-              body: 'This is the body of the toast, glad you are here. And it is a little bit longer and should wrap around the card.'
-            })
-          }
-        >
-          Success Toast
-        </Button>
-        <Button
-          onClick={() =>
-            triggerToast({
-              status: 'info',
-              header: 'Serving up some info',
-              body: 'This is the body of the toast'
-            })
-          }
-        >
-          Info Toast
-        </Button>
+            </PrescribeProvider>
+          </RecentOrders>
+        </DraftPrescriptionsProvider>
       </Client>
     </div>
   );
 };
 
 export default App;
+
+// const draftPrescriptions: DraftPrescription[] = [
```

Comment:
can we get rid of this?

Reply:
oh wait we should add all of that other stuff back, I removed it to make it easier to look at the app page